src/components/directory-tree-view.js

```diff
@@ -27,6 +27,7 @@ const useStyles = makeStyles((theme) => ({
         padding: theme.spacing(0.5),
     },
     treeItemRoot: {
+        userSelect: 'none',
```

Comment:
why do we need this ?

Reply:
userSelect=none is to disable the ability to select text from the tree. This is useless, and it happens often by mistake. 
(validated with PO)
![select-tree-view-orange](https://user-images.githubusercontent.com/107846716/176190763-976ff1ae-6437-431d-8799-cc5c0247f8eb.gif)